docs/feature_oled_driver.md

```diff
@@ -235,15 +263,15 @@ So those precalculated arrays just index the memory offsets in the order in whic
 ## OLED API
 
 ```c
-// OLED rotation enum values are flags
+// OLED Rotation enum values are flags
```

Comment:
I recopied the relevant chunk of the .h file. That's where the other changes came from.